VideoToImage/videotoimage.py

```diff
@@ -0,0 +1,41 @@
+import botocore
+import subprocess as sp
+import os
+import boto3
+import datetime
+
+#No. of frames per second
+FRAMES = int(os.environ.get('FRAMESPERSEC'))
+
+def convertVideoToImage(srcKey, srcBucket):
+    s3 = boto3.resource('s3')
+    localFilename = '/tmp/{}'.format(os.path.basename(srcKey))
+    #Download video to local
+    try:
+        s3.Bucket(srcBucket).download_file(srcKey, localFilename)
+    except botocore.exceptions.ClientError as e:
+        if e.response['Error']['Code'] == "404":
+            print("The object does not exist.")
+        else:
+            raise
+    #Chop video into images
+    cmd='ffmpeg -i '+str(localFilename)+' -r '+str(FRAMES)+' img_%04d.jpg'
+    sp.call(cmd,shell=True)
+
+    #Upload file to s3
+    for imgFile in os.listdir("./"):
```

Comment:
so we are writing to the local folder? Might want to write these to tmp also in an /tmp/output folder or something like that. Also make this configurable.